### ISSUE TITLE
Copycat/Clone of Spotube on Google Play

### ISSUE THREAD

**Issue by Poussinou** [KRTirtho/spotube#448]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

Hi,

Somebody just found this copycat of Spotube on Google Play : https://play.google.com/store/apps/details?id=oss.krtirtho.spoasde from [this developer](https://play.google.com/store/apps/developer?id=Schnupfeld). It's full of ads.

### Suggested solution

I'm not familiar with BSD-4-Clause License, and I don't know if this copycat respects the license or not, but you can follow [this procedure](https://github.com/TeamNewPipe/NewPipe/issues/539#issuecomment-1399222536) if you want to send a DMCA takedown request to Google.

**Comment by KRTirtho** [comment]
Following https://github.com/TeamNewPipe/NewPipe/issues/539#issuecomment-1399222536 and https://github.com/TeamNewPipe/NewPipe/issues/539#issuecomment-1419844687 I've made an official complaint about the whole copycat situation. Now let's hope, **oss.krtirtho.spoasde** "Copycat" will be taken down for violating Clause 3 and 4 of BSD-4-Clause "Original" or "Old" License

## Follow the below steps to submit your complaints too. Because only my complaint will never create a big attraction. It's now time to unite and fightback :muscle:

### Steps:
1. Go to **https://support.google.com/googleplay/android-developer/contact/takedown** to file a complaint
2. For the field **"Name"** write: `Spotube`
3. For the field **"Application package name"** write: `oss.krtirtho.spoasde`
4. For the field **"Reason for flagging"** select: `Other objection`
5. For the field **"Please explain briefly why you flagged the app"*** please use:

    ```txt
    The OFFICIAL Spotube app is a free and open source application licensed under the BSD-4-Clause "Original" or "Old" license, whose source code is available on GitHub at https://github.com/KRTirtho/Spotube. The app on Play Store, oss.krtirtho.spoasde, is an UNAUTHORIZED COPY of the official Spotube app.
    Because BSD-4-Clause License requires any advertising materials mentioning features or use of Spotube to display "This product includes software developed by Kingkor Roy Tirtho." (See line 3 to 4 of the license at https://github.com/KRTirtho/spotube/blob/master/LICENSE#L9-L10) but it doesn't display any such notice.
    Moreover, BSD-4-Clause license disallows the usage of the name Spotube or author's name "Kingkor Roy Tirtho" without written permission to endorse or promote products derived from Spotube (See line 11 at https://github.com/KRTirtho/spotube/blob/master/LICENSE#L11) but publisher Schnupfeld (https://play.google.com/store/apps/developer?id=Schnupfeld) is using the name "Spotube"
    ```
6. Use the attached images for **"Attach a screenshot to help troubleshoot your issue"** field
    - Additionally, upload a screenshot of `oss.krtirtho.spoasde` in the Playstore app's listing (search result)
7. For the field, **"Is the application still available on Google Play?"** select: `Yes`
8. Finally, hit submit and hope it'll get taken down :crossed_fingers: 

#### Images to upload in the report description
![icon](https://user-images.githubusercontent.com/61944859/225382880-60dd628b-bcfb-4183-9bc7-71c2c9112e91.png) ![screenshot1](https://user-images.githubusercontent.com/61944859/225382892-2c597da2-1dfc-4e92-82ef-e5f0a91010e8.png) ![screenshot2](https://user-images.githubusercontent.com/61944859/225382898-ce70f61a-3f0b-44b9-b032-437fad069fab.png) ![screenshot3](https://user-images.githubusercontent.com/61944859/225382900-7239f646-6342-44ca-ae84-6cbd81a40ca4.png)

**Comment by owencz1998** [comment]
Submitted a report let's hope for the best 🤞

**Comment by Poussinou** [comment]
@KRTirtho Please don't hesitate to report it here if you find new copycats. Most of the time, that kind of "developers" have many copycats for many FLOSS apps (that's the case here). If we report them all, their account can be taken down.

**Comment by KRTirtho** [comment]
Definitely. We've to get rid of these stealers. Such people are dangerous to the FOSS community, and it's a shame to call them "developers"
If anyone finds any Copycat (Illegal clone) of Spotube available in Play store, or in any software distribution platform that was modified but using the name "Spotube" to identify itself without displaying "This product includes software developed by Kingkor Roy Tirtho." please report this here. We'll try our best to take down the app

Thanks to everyone who are helping :heart:

**Comment by KRTirtho** [comment]
### Good news!!!
Finally, copycat/illegal clone of Spotube on Play Store was removed from the listing :partying_face: 
Thanks to everyone who has helped Spotube :rose:  :muscle:

**Comment by Harfeur** [comment]
Hi! Please update the wiki to remove this 😉

**Comment by KRTirtho** [comment]
Thanks for the reminder. Almost forgot about that 😅

**Comment by shuvashish76** [comment]
@KRTirtho https://play.google.com/store/apps/details?id=oss.krtirtho.spotube is it the official one?

I'm actually surprised how the official app is on PlayStore despite having Piped & JioSaavn features. I guess they didn't notice the app yet? IMO better to remove it than get kick out in future.
After [BlackHole](https://gitlab.com/fdroid/fdroiddata/-/issues/3219) went into black hole probably due to piracy issues (it was using jiosaavn? to fetch music), Spotube getting more popularity.

**Comment by KRTirtho** [comment]
Yes, it's the official one.

> I'm actually surprised how the official app is on PlayStore despite having Piped & JioSaavn features. I guess they didn't notice the app yet? IMO better to remove it than get kick out in future.
After [BlackHole](https://gitlab.com/fdroid/fdroiddata/-/issues/3219) went into Blackhole probably due to piracy issues (used to fetch music from JioSaavn), Spotube getting more popularity.

I guess the author somehow was making revenue through appstores/playstore. It had a paid and a free version (later author removed the free version). So, even though it was OSS, it was technically commercialized. That's why I think it caught the attention of the corp.

So I always try to avoid making money through this app. Donations are considered non-profit so they're welcome.

**Comment by shuvashish76** [comment]
> I guess the author somehow was making revenue through appstores/playstore.

IIRC they never published BlackHole on PlayStore.

> So I always try to avoid making money through this app. Donations are considered non-profit so they're welcome.

Thanks but it doesn't matter if you're non-profit or not. It's against Google's policy that's why we see many YouTube clients on F-Droid but none of them available on PlayStore.
